### DISCUSSION TITLE
I use getMyFeesEstimates() to get the product fee of the SKU. After submitting the API request, the returned result is empty, without any data, and there is no mark of success

### DISCUSSION THREAD

**Discussion by fengruilk** [amzn/selling-partner-api-models#3530] [General]
I use getMyFeesEstimates() to get the product fee of the SKU. After submitting the API request, the returned result is empty, without any data, and there is no mark of success

**Comment by sungolivia** [comment]
I think it would help other fellow community folks and us if you can share your request and responses? 😄

**Comment by fengruilk** [comment]
Here is my request code
`$listingPrice = new \SellingPartnerApi\Model\FeesV0\MoneyType(array('currency_code' => 'USD','amount' => 3.92));
 $price = new \SellingPartnerApi\Model\FeesV0\PriceToEstimateFees(array('listing_price' => $listingPrice));
 $feesEstimateRequest = new \SellingPartnerApi\Model\FeesV0\FeesEstimateRequest(array('marketplace_id' => 'ATVPDKIKX0DER', // Germany
            'is_amazon_fulfilled' => false,
            'price_to_estimate_fees' => $price,
            'identifier' => 'd'
        ));
        $id_type = new \SellingPartnerApi\Model\FeesV0\IdType('SellerSKU');
        $body = new \SellingPartnerApi\Model\FeesV0\FeesEstimateByIdRequest(array('fees_estimate_request' => $feesEstimateRequest));
        $body->setIdType($id_type);
        $body->setIdValue('211-xgj-3-388');
try {
            $result = $apiInstance->getMyFeesEstimates(array($body));
            print_r($result);
        } catch (Exception $e) {
            echo 'Exception when calling FeesV0Api->getMyFeesEstimates: ', $e->getMessage(), PHP_EOL;
        }`
The following is the request returned to the data
`Array
(
    [0] => SellingPartnerApi\Model\FeesV0\FeesEstimateResult Object
        (
            [container:protected] => Array
                (
                    [status] => 
                    [fees_estimate_identifier] => 
                    [fees_estimate] => 
                    [error] => 
                )

        )

)`

**Reply by iololife** [reply]
I'm also interested in an answer to this question as I'm running into a similar error using Java.

**Comment by gokigoks** [comment]
I'm still getting this issue. 

Small sample ASINs: B07733Q47J , B0773FSRJP, B0773CY498, B0773HBGWB. 

### Response:

```
{
   "payload":{
      "FeesEstimateResult":{
         "Status":"ClientError",
         "FeesEstimateIdentifier":{
            "MarketplaceId":"ATVPDKIKX0DER",
            "SellerId":"XXXXXXXXXXXXX",
            "IdType":"ASIN",
            "IdValue":"B07733Q47J",
            "IsAmazonFulfilled":true,
            "PriceToEstimateFees":{
               "ListingPrice":{
                  "CurrencyCode":"USD",
                  "Amount":89.26
               }
            },
            "SellerInputIdentifier":"B07733Q47J"
         },
         "Error":{
            "Type":"Sender",
            "Code":"InvalidParameterValue",
            "Message":"There is an client-side error. Please verify your inputs.",
            "Detail":[]
         }
      }
   }
}
```

I hope we can get more details from the error response on which field is causing the issue.
